pyuvdata/uvdata/fhd.py

```diff
@@ -618,9 +618,10 @@ def read_fhd(
         self._set_phased()
         self.phase_center_ra_degrees = float(obs["OBSRA"][0])
         self.phase_center_dec_degrees = float(obs["OBSDEC"][0])
+        self.phase_center_frame = "icrs"  # TODO: Confirm this
```

Comment:
Alright, after a fair bit of digging, I think I _finally_ got an answer to this question. So UVFITS does not have a default, per se, but does typically write the coordinate system to the header value `RADESYS` (or alternatively, `RADESYSA`) to determine what it should be. Failing that, looking at some [AIPS documentation](http://www.aips.nrao.edu/cgi-bin/ZXHLP2.PL?REGRD), it seems that the default is to assume FK4 if `EPOCH` is 1950.0, otherwise if any value is given, to assume that is FK5. Presumably, if one were to find no value for `EPOCH`, one might assume that the frame should be ICRS.

I checked a few UVFITS files, and I do indeed see `RADESYS` in nearly all of them. On a hunch, I decided to check the .sav files used for the FHD unit tests (`1061316296_obs.sav`), and as luck has it, in the `astrometry` dict (rec?), there is a field called `RADECSYS`, which just so happens to have the value 'FK5'. Minor mystery solved!

Given that we've identified the right keyword in FHD, I think I'll adjust the above to look for that now, and will make some corresponding changes in the UVFITS reader.